### ISSUE TITLE
Subtitle default option per user

### ISSUE THREAD

**Issue by cesalo** [jellyfin/jellyfin-androidtv#72]
The option to setup a default subtitle option for all movies/tv-shows is not present like in the android app or in browser. This means that I have to enable for each movie and episode by episode.

**Comment by plougsgaard** [comment]
@thornbill would you accept a PR on this?

I'm contemplating putting an input field in Settings with comma-separated subtitle languages the app could use to pre-set a subtitle with.

`"fr,eng"`

Prefers French if possible, then English if that's not found.

**Comment by thornbill** [comment]
A text input would probably not be the best experience on a device with a remote as an input. I would prefer a dropdown field like what is used in web. No objections at all to allowing multiple selections though!

**Comment by plougsgaard** [comment]
I've had a look and it seems that if I choose the right index here it will work:

https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/playback/PlaybackController.java#L622

It should also respect the `isForced` flag I reckon, if `response.getMediaSource().getDefaultSubtitleStreamIndex()` doesn't find anything. 

Do you have any suggestion on getting the list of languages? The web client uses something to the effect of `apiClient.getCultures()` - is this available here?

**Comment by keithdok** [comment]
Any update on this?

**Comment by nielsvanvelzen** [comment]
We might look at this for 0.13 (after the next major release) as that release focusses on improving the playback of media.

**Comment by labaland** [comment]
Is this added now? I can not find it on my shield tv!!  :)

**Comment by nielsvanvelzen** [comment]
No. We're still working on 0.12

**Comment by labaland** [comment]
> No. We're still working on 0.12

Ok thx! Just switched from emby. When i try to change audio track/language app always craches.

**Comment by SonicPotions** [comment]
I would love if this could be set per TV-Show as well.
In Plex at least it can be set before starting the episode, way better than changin audio and subtitle language when the episode already started.
a per tv-show setting would be even better. I'm not a fan of global settings, as it is very mixed depending on the movie/show what language I select. global + per show setting that overrides the global would be best.

**Comment by JVT038** [comment]
Has this been added yet? This would be very useful to me!

**Comment by plopingo** [comment]
No. I can't choose a default subtitle . same for default audio.

**Comment by grayum** [comment]
Same here with version 0.15.0-beta-4

**Comment by plopingo** [comment]
> Same here with version 0.15.0-beta-4

I hope they can add this box on android tv 

![image](https://user-images.githubusercontent.com/2069957/207848935-809af251-fa0b-484f-9401-edb1a45a1dee.png)
